src/layouts/Layout.vue

```diff
@@ -57,6 +57,12 @@
                                 </router-link>
                             </li>
 
+                            <li>
+                                <router-link to="/incident-history" class="dropdown-item" :class="{ active: $route.path.includes('manage-incident-history') }">
+                                    <font-awesome-icon icon="bullhorn" /> {{ $t("Incident History") }}
+                                </router-link>
+                            </li>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rohankokkulabito/uptime-kuma/pull/1#issuecomment-2265233525>#1bce46</a><span> - 08/02/2024, 12:20 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The use of '$route.path.includes('manage-incident-history')' for determining the active class is not robust. If the route path contains 'manage-incident-history' as a substring, it will incorrectly mark the link as active. This can lead to unexpected behavior if there are other routes with similar substrings.
</div>
<div id="fix">
<b>Fix:</b> Use a more precise method to determine if the current route matches the intended path. Vue Router provides named routes which can be used for this purpose.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -61,7 +61,7 @@
                              <router-link to="/incident-history" class="dropdown-item" :class="{ active: $route.name === 'manage-incident-history' }">
                                  <font-awesome-icon icon="bullhorn" /> {{ $t("Incident History") }}
                              </router-link>
                          </li>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged